src/TypeSpec.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -805,6 +801,11 @@ export function getInputType(
     }
 
     function getInputTypeForUnion(union: Union): InputUnionType | InputType {
+        var clientType = getClientType(context, union);
+        if (clientType.kind === "enum" && clientType.isFixed === false) {
```

Comment:
Why does we check `isFixed` here? Can it be a Fixed enum if it defined as Union? If so, shall we emit fixed enum and do not ignore?

Reply:
> Can it be a Fixed enum if it defined as Union

Yes.

> shall we emit fixed enum and do not ignore

Fixed enum is next step. We only handle extensible enum in this PR.

Reply:
I don't quite understand. So we can use Union to define a fixed enum, but now we will not support this feature, and return Union as type as before?

Reply:
Yes, if you define `"a"|"b"`, it is still union.